pages/docs/community/styleguide/inclusive-language.md

```diff
@@ -0,0 +1,57 @@
+# Style Docs - Inclusive Language:
```

Comment:
```suggestion
# Inclusive Language
```
you can remove style docs in the heading as it is already a part of it. No need to declare exclusively